server/ormconfig.json

```diff
@@ -1,24 +0,0 @@
-{
```

Comment:
Why so? You need to either define connection params here, through environment variables, or in `TypeOrmModule` as `.forRoot()` param, thus it will be suitable for deployment and QA.

Reply:
We decided to use env (.env file)variables to make process identical on  Heroku and locally. I've made changes on Wiki (set-up local DB page)

Reply:
ok, gotcha